.github/operators_post_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+
+# Print basic outputs for debug
+oc get pods -n openstack
+
+# Create clouds.yaml file to be used in further tests.
+mkdir -p ~/.config/openstack
+cat > ~/.config/openstack/clouds.yaml << EOF
+$(oc get cm openstack-config -n openstack -o json | jq -r '.data["clouds.yaml"]')
+EOF
+export OS_CLOUD=default
+export OS_PASSWORD=12345678
```

Comment:
instead of hard coding passwords here, shouldn't we fetch this from the secret

Reply:
something like `oc get secret $(oc get keystoneapi keystone -o json | jq -r .spec.secret) -o json | jq -r .data.$(oc get keystoneapi keystone -o json | jq -r .spec.passwordSelectors.admin) | base64 -d`

Reply:
Thanks Martin, done.